README.md

```diff
@@ -42,16 +42,16 @@ Before you can use the MCP server, you need to:
    # Option A: If you have the CLI installed globally
    npm install -g sanity
    cd /path/to/studio
-   SANITY_CLI_SCHEMA_STORE_ENABLED=true sanity schema deploy
+   sanity schema deploy
 
    # Option B: Update your Studio
    cd /path/to/studio
    npm update sanity
-   SANITY_CLI_SCHEMA_STORE_ENABLED=true npx sanity schema deploy
+   npx sanity schema deploy
```

Comment:
`sanity deploy` also deploy schemas

I _might_ be worth noting that `sanity schema deploy` requires a deploy token.
If you are a dev and use `sanity login` this is not a problem, but on CI  you will have to

`SANITY_AUTH_TOKEN=<toksanity schema deploy`

Reply:
Updated